src/ssa.c

```diff
@@ -401,6 +401,49 @@ void build_rdf()
     free(args);
 }
 
+void users_add_tail(insn_t *i, var_t *var)
+{
+    user_t *u = calloc(1, sizeof(user_t));
```

Comment:
Add handler when calloc failed (i.e. u == NULL)

Reply:
OK, thank you for your reminder.